radlab-ui/webapp/src/pages/api/regions.ts

```diff
@@ -0,0 +1,55 @@
+import { NextApiRequest, NextApiResponse } from "next"
+
+import axios from "axios"
+import { generateAccessToken } from "@/utils/api"
+import { envOrFail } from "@/utils/env"
+
+const projectId = envOrFail(
+  "NEXT_PUBLIC_GCP_PROJECT_ID",
+  process.env.NEXT_PUBLIC_GCP_PROJECT_ID,
+)
+
+const getRegions = async (_: NextApiRequest, res: NextApiResponse) => {
+  const token = await generateAccessToken()
+  const data = await axios({
+    method: "GET",
+    url: `https://compute.googleapis.com/compute/v1/projects/${projectId}/regions`,
+    headers: {
+      Authorization: `Bearer ${token}`,
+    },
+  })
```

Comment:
This should be wrapped in a `try/catch` and return something the frontend can use in case of an error.

Reply:
This should also be named `response` and not `data`.